### PR TITLE
created the yaml file :D

### PR DESCRIPTION
**Pull request by joseivanpo**
Created the yaml file and it's ready to merge

### CODE REVIEW

**Comment by AlgusDark** [comment]
👍

**Comment by castrosalazr** [comment]
👍

File: _pages/joseivanpo.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+thubHandle: joseivanpo
```
Comment (brianamarie) [inline _pages/joseivanpo.yaml]: Hi @joseivanpo! There are some small syntax errors that are making the build fail.

Make sure that in line 2, it says `githubHandle`. 

In lines 3 and 4, make sure there is no space at the beginning of the lines. 

After line 5, make sure there is a line break.

Let me know if you want or need any help. 😄

**Comment by brianamarie** [comment]
👋 By the way, if this pull request hasn't seen any updates in a few days, one of the GitHub trainers will come around and clean it up to merge it in so we can keep the repository up to date. In the meantime, let me know if you want or need any help. 😄

**Comment by brianamarie** [comment]
@joseivanpo  it looks like the tests are failing because your web page no longer exists or pages has not been enabled on the repository. If you have created your repository, please go to the repository **settings** and scroll down to the **pages** section. Using the drop-down, select the **master** branch to render your website. When you have finished this, your web page will display at the link shown at the top of the pages section. I am going to go ahead and close this pull request since your page is not currently enabled. If you would like to finish this activity, please feel free to re-open this pull request after you have enabled pages on your repository.

**Pull request closed without merging**